javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -19,8 +19,11 @@
 package com.github.javaparser.ast.body;
 
 import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.Modifiers;
+import com.github.javaparser.ast.NodeWithModifiers;
 
 import java.lang.reflect.Modifier;
```

Comment:
Then I guess 'abstract' is not a modifier too... :P
I mean they could as well specify abstract methods on an abstract class as just not having a body but they chose to add a modifier keyword. They did the same with default methods on interfaces: they added a modifier keyword.
Well that is the purpose of a modifier.

Our role in parsing java sources is to model the syntax. Following your point of view, we could as well drop the isDefault boolean in MethodDeclaration and let the user deal with testing if there is a body or not. But then why not do the same for abstract methods in abstract classes.
Nah 'default' is definitely a modifier and does not deserve its own boolean flag.

I even think that InitializerDeclaration should use modifiers even if only one (namely 'static') is possible.